pages/api/book/event.ts

```diff
@@ -263,9 +263,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const seed = `${users[0].username}:${dayjs(req.body.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
+  const eventNameObject: EventNameObjectType = {
```

Comment:
Explicit typing is not needed here, you can pass eventNameObject to the getEventName function and it will complain without `: EventNameObjectType` if the type is incompatible with the function.

Reply:
This applies to all instances of applying this type.